Assets/MRTK/Providers/Oculus/XRSDK/OculusXRSDKDeviceManager.cs

```diff
@@ -264,6 +267,13 @@ protected void UpdateHand(OVRHand ovrHand, OVRSkeleton ovrSkeleton, OVRMeshRende
             if (ovrHand.IsTracked)
             {
                 var hand = GetOrAddHand(handedness, ovrHand);
+
+                bool showHandMesh = handTrackingProfile.EnableHandMeshVisualization;
+                bool showHandJoints = handTrackingProfile.EnableHandJointVisualization;
+
+                ovrMeshRenderer.enabled = showHandMesh;
+                hand.Visualizer.GameObjectProxy.SetActive(showHandJoints);
```

Comment:
nit: should this be handled inside the OculusHand class, since we're asking it for its Visualizer here?